src/scaffold/ScaffoldView.tsx

```diff
@@ -162,7 +162,7 @@ const SecondPage: React.FC<PageProps> = ({ onPrevious }) => {
                         <p className="extraItemLabel">Version</p>
                       </VSCodeTextField>
                     </div>
-                    <div className={`extraItem ${options.repoType === 'Piral' ? 'hide' : ''}`}>
+                    <div className={`extraItem ${repoType === 'Piral' ? 'hide' : ''}`}>
```

Comment:
Please change the available repo types to `piral` and `pilet`. Always lowercase.